views/layouts/main.handlebars

```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="utf-8">
     <title>Google STEP App</title>
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=API_KEY_HERE&callback=initMap"></script>
```

Comment:
Should we get an api key before pushing this?

Reply:
That's probably a good idea. 👍